Homework2/CurrentValueCalc.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+
+namespace Homework2
+{
+    public class CurrentValueCalc : ICurrentValueCalc
+    {
+        private int _nominal;
+        private double _priceTrade;
+        private double _priceOrigin;
+
+        public int Nominal
+        {
+            get { return _nominal; }
+            set
+            {
+                try
+                {
+                    if (value > 0)
+                    {
+                        _nominal = value;
+                    }
+                    else
+                        throw new Exception("Number of shares must be bigger than 0");
+                }
+                catch (Exception e)
+                {
+                    Console.WriteLine(e.Message);
+                }
```

Comment:
try/catch must be moved to another location where CurrentValueCalc will be used. In this case, the class user can decide how exception should be handled and whether it should be a message to the console or a message box or anouther action, such as requesting another value from the user.